src/main/java/mate/academy/rickandmorty/service/SaveCharacters.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.rickandmorty.service;
+
+import jakarta.annotation.PostConstruct;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.external.CharacterResponseDataDto;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class SaveCharacters {
```

Comment:
Verbs shouldn't be used in class names
```suggestion
public class CharactersInitializer {
```